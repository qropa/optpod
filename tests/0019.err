[DATA] sigma = 5780.0
[DATA] l = 37584
[DATA] occ = 87
[DATA] score = 0.952
Time = 2.8099913597106934
