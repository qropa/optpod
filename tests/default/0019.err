[DATA] sigma = 5780.0
[DATA] l = 37584
[DATA] occ = 87
[DATA] score = 0.952
Time = 2.8087213039398193
