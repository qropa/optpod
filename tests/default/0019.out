37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 0 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 1 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 0 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 1 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 1 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 0 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 0 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 1 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 1 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 1 U 20
22 1 U -1
23 0 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 0 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 1 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 1 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 0 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 1 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 1 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 0 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 0 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 1 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 0 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 0 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 1 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 0 U 19
21 1 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 0 U 24
26 0 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 1 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 0 U 19
21 0 U 20
22 1 U -1
23 0 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 1 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 0 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 1 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 1 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 1 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 1 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 0 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 0 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 0 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 0 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 1 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 0 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 1 U 1
3 1 U 2
4 1 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 1 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 1 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 1 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 0 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 1 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 0 U 27
29 0 U -1
30 1 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
37
0 0 U -1
1 1 U 0
2 0 U 1
3 1 U 2
4 0 U 3
5 1 U 4
6 0 U 5
7 1 U 6
8 0 U -1
9 0 U 8
10 1 U 9
11 0 U 10
12 0 U 11
13 1 U 12
14 1 U 13
15 0 U -1
16 1 U 15
17 0 U 16
18 0 U 17
19 1 U 18
20 1 U 19
21 0 U 20
22 1 U -1
23 1 U 22
24 0 U 23
25 1 U 24
26 1 U 25
27 1 U 26
28 1 U 27
29 0 U -1
30 0 U 29
31 1 U 30
32 1 U 31
33 0 U 32
34 0 U 33
35 0 U 34
36 0 U 35
